data/fro/Zink_1984_Cleriadus/stemma.gv

```diff
@@ -0,0 +1,34 @@
+digraph {
+
```

Comment:
I think we are missing the indication of the manuscripts that are hypothetical/lost (with `color="grey"`). Am I correct in assuming these are the only ones ?

```gv

O[color="grey"];
x[color="grey"];
y[color="grey"];
xprimeprime [label="x''", color="grey"];
xprime [label="x'", color="grey"];
```
?